content/zh/docs/tasks/observability/logs/otel-provider/snips.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
```

Comment:
I think we should not sync `*.sh` to other language folder, it's meaningless, @ericvn WDYT?